seed/views/v3/tax_lot_properties.py

```diff
@@ -163,7 +163,11 @@ def export(self, request):
         # force the data into the same order as the IDs
         if ids:
             order_dict = {obj_id: index for index, obj_id in enumerate(ids)}
-            data.sort(key=lambda x: order_dict[x['id']])  # x is the property/taxlot object
+            if view_klass_str == 'properties':
+                view_id_str = 'property_view_id'
+            else:
+                view_id_str = 'taxlot_view_id'
+            data.sort(key=lambda inventory_obj: order_dict[inventory_obj[view_id_str]])
```

Comment:
I did a double check in `TaxLotProperty.get_related()`, which is where `data` is coming from, and it seems we're guaranteed these view IDs so this should be OK

Reply:
That sounds right - thanks for double checking 💯